networking/src/index.ts

```diff
@@ -1,7 +1,7 @@
 // created from 'create-ts-index'
 
-export * from './AWSMobileApi'
 export * from './ApiClient'
+export * from './AWSMobileApi'
```

Comment:
I pushed a commit in `dev` to add this to `.gitignore` list so it doesn't show up. Feel free to delete this file locally and merge the `dev`. This file is auto-generated and no need to be in the source control

Reply:
sweet!